src/stdlib.c

```diff
@@ -251,6 +251,23 @@ Array.prototype.lastIndexOf = function(a, b) {
   return -1;
 };
 ));
+
+  v7_exec(v7, &res, STRINGIFY(
+    Array.prototype.reduce = function(a, b) {
+      var f = 0; var r = b; var t = this;
+      if (typeof(a) != 'function') {
+        throw new TypeError('oops');
```

Comment:
Done.
Also, increased a limit in `i_populate_local_vars()` to make jquery compile